charts/add-ons/grafana/values.yaml

```diff
@@ -0,0 +1,4 @@
+
+# must be present to avoid access to non-existing element of the Values tree
+proxy:
+  resources:
```

Comment:
This empty value is generating some errors when I try to override the grafana properties in the main `values.yaml`. And now that you put in place the checks for the existence of these properties, we can leave this file totally empty as it was before.